src/server/map.ts

```diff
@@ -83,6 +83,10 @@ export class Map {
     this.tileUpdate(coords);
   }
 
+  isInBounds(coords: Coords): boolean {
+    return coords.x > 0 && coords.x < this.width && coords.y > 0 && coords.y < this.height;
```

Comment:
Aren't x = 0 and y = 0 valid coords?
So, shouldn't this read `return coords.x >= 0 && coords.x < this.width && coords.y >= 0 && coords.y < this.height;` instead?